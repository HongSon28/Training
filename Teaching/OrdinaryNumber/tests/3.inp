7
53
51
77
6
15
52
30
