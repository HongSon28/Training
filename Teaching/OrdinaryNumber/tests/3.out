13
13
16
6
10
13
11
